cibuildwheel/__main__.py

```diff
@@ -99,7 +99,7 @@ def main():
         traceback.print_exc(None, sys.stderr)
         exit(2)
 
-    skip = BuildSkipper(skip_config)
+    selection = BuildSelector(build_config, skip_config)
```

Comment:
BuildSelector seems like a reasonable name to me. `selection` is a little confusing though? Maybe just call it `build_selector`? open to other ideas though

Reply:
Myeah, I liked how `skip(...)` read, but couldn't really reproduce that

Reply:
The other option is to keep this as `BuildSkipper`, btw, and just add the extra argument of `build_config`? Depending on your point of view, this class is still determining what builds to skip.

Reply:
BuildSkipper feels like it relates too closely with CIBW_SKIP. I like it the way it is, I think, just the `selection(identifier)` is a bit too vague.